clean-up-migrations.md

```diff
@@ -0,0 +1,126 @@
+# Clean up your migrations!
+
+Database migrations are a great way to safely update your database schema,
+and so it's very useful for production where you do not want to lose the data
+you have. If. you want to know more about migrations - SymfonyCasts has a
+[screencasts](https://symfonycasts.com/screencast/symfony-doctrine/migrations)
+about it, check it out!
+
+But here, I will give you a tip on how to clean up your migrations correctly.
+First of all, this is mostly for local development, because usually it's not
+important how many migrations you have on production - all those migrations
+were already executed.
+
+But in local development it's also a good idea to create the database and run
```

Comment:
```suggestion
But while developing locally, to set things up, we recommend creating your database and
then running the `doctrine:migrations:migrate` command so that you have exactly what's
on production, including a `migrations_versions` table. This is better than running the
`doctrine:schema:update` command, which then makes it hard to test new migrations.
```